src/routes/events/index.svelte

```diff
@@ -37,15 +35,15 @@
 
 <Spacing --min="100px" --med="175px" --max="200px" />
 
-<h2 class="size-lg headers">Upcoming Events 📅</h2>
+<h2 class="size-lg headers">Upcoming events 📅</h2>
```

Comment:
#445

Reply:
how r u just gonna revert @karnikaavelumani only pull request like that

Reply:
BRUH FR LOOK AT WHAT HE DID @Peekoe ALL OUR EFFORT

Reply:
How dare they remove the
![183.jpg](https://user-images.githubusercontent.com/56566188/165013486-be86e1b0-2af7-43d0-b376-4752d1481e29.jpg)

Reply:
It doesn't follow how the rest of our headers are

Reply:
counterproposal: make the rest of the headers match

Reply:
@jaasonw check discord poll on issue